### ISSUE TITLE
unify integration-tests

### ISSUE THREAD

**Issue by laeubi** [eclipse-tycho/tycho#122]
Currently we have:

- tycho-its
- tycho-extras-its
- some 'its' in the bundles

it would be good to unify this a bit and have only one tycho-its module that covers all cases. it would also be good to structure the projects a bit, currently we have a mixture of bugnames, functional prefixes and some arbitrary names.

Ideally we would have a structure like this:
```
- projects
-- compiler-plugin
--- testcase name
-- package-plugin
-- pomless-extension
-- ....
```

We could also have a special folder "bugs" where we collect reproducers for a given bug by users, if the bug is identified we move this to the actual place and give it a sensible name

**Comment by akurtakov** [comment]
This is all done AFAICT.

**Comment by laeubi** [comment]
> This is all done AFAICT.

Not at all, look at the project structure and you will still see the rather random structure.

**Comment by akurtakov** [comment]
Would you please list these?

**Comment by laeubi** [comment]
Just compare

> Ideally we would have a structure like this
> ...

with what you find here https://github.com/eclipse-tycho/tycho/tree/main/tycho-its/projects

**Comment by akurtakov** [comment]
So what's left is tycho-its internal projects restructuring while the move from other "its" tests is done.

**Comment by laeubi** [comment]
> So what's left is tycho-its internal projects restructuring while the move from other "its" tests is done.

Yes if I remmber correctly I have now migrated all its to the central place.
